### PR TITLE
Integrate with @comicrelief/eslint-config (#83)

### LINKED ISSUE #83: Integrate with @comicrelief/eslint-config.
The repository should take advantage from Comic Relief's the shared Eslint config.

### PR DESCRIPTION
**Pull request by dipasqualew**
In order to not introduce regression on highly complex functions, some of the rules have been disabled, with the aim of resolving disabled rules one by one in future releases.

Added the `yarn lint` command.

Closes #83

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,20 +1,6 @@
...
-    "no-param-reassign": "off",
-    "react/destructuring-assignment": "off",
-    "flowtype/no-types-missing-file-annotation": "off",
-    "max-len": "off",
-    "import/no-cycle": "off",
-    "no-else-return": "off",
-    "class-methods-use-this": "off",
-    "consistent-return": "off"
-  }
+  "parser": "babel-eslint"
```
Comment (lebaz20) [inline .eslintrc:5]:
```suggestion
  "parser": "@babel/eslint-parser"
```

**Review by lebaz20 — CHANGES_REQUESTED** [review]
Commit messages need to trigger `semantic-release` process.

**Comment by lebaz20** [comment]
@dipasqualew you can edit commit messages to include `fix:` for instance and push force may be?

**Comment by dipasqualew** [comment]
I don't think it's required because this change doesn't provide any new feature nor change in functionality. It's merely a dev update, so I don't think there's anything to release

**Review by lebaz20 — DISMISSED** [review]

**Review by lebaz20 — APPROVED** [review]

**Pull request merged**